lib/driver.js

```diff
@@ -747,12 +747,18 @@ class XCUITestDriver extends BaseDriver {
           `to avoid conflicts and speed up the building process`
       );
     }
+
+    if (this.opts.usePreinstalledWDA && !this.isRealDevice()) {
+      throw new Error(`'usePreinstalledWDA' capability must be against a real device`);
```

Comment:
...capability is only supported for real devices. Use `appium:useXctestrunFile` one  if you need to init WebDriverAgent without xcodebuild on Simulators

Reply:
`useXctestrunFile` uses xcodebuild command, but it uses an existing built package. So it is not equivalent to usePreinstalledWDA. (so it is available for real devices as well)
Maybe useXctestrunFile is similar to usePrebuiltWDA capability but more portable

Reply:
so there is no direct equivalent for simulators?

Reply:
correct, I haven't seen such a method yet. i haven't investigated well, but I wondered if we could use simctl something like https://stackoverflow.com/questions/33633882/run-xctest-bundle-with-simctl , but maybe need additional argument etc